scrapi/base/schemas.py

```diff
@@ -10,6 +10,8 @@
     oai_process_contributors
 )
 
+CONSTANT = lambda x: lambda *_, **__: x
```

Comment:
I just figured this would prevent us from having to keep CONSTANT in sync with however we end up processing functions.